lametro/views.py

```diff
@@ -251,6 +254,12 @@ def delete_submission(request, event_slug):
     return HttpResponseRedirect('/event/%s' % event_slug)
 
 
+def delete_event(request, event_slug):
```

Comment:
You definitely want to make sure this view cannot be accessed by non-admin users. Can you propose a solution?